### DISCUSSION TITLE
[FR] CLI Command to list currently installed plugins with handles & versions

### DISCUSSION THREAD

**Discussion by ademers** [craftcms/cms#8916] [Ideas]
### Description

It would be useful to be able to display a list of installed plugins with handles & versions. Also, the currently installed version of Craft.

Something similar to the existing `./craft update/info` command, but not just for stuff with available updates.

It would also make a handy companion to the existing plugin commands, since they require knowledge of a plugin's handle.

```
- plugin                                    Manage plugins
    plugin/disable                          Disables a plugin.
    plugin/enable                           Enables a plugin.
    plugin/install                          Installs a plugin.
    plugin/uninstall                        Uninstalls a plugin.
```
Motivation for FR: https://discordapp.com/channels/456442477667418113/575387375610626063/751107023780839444

Thanks for you consideration!

**Comment by lindseydiloreto** [comment]
This sounds like a great idea! Maybe `plugin/info` or `plugin/list` to see more comprehensive details from the command line?

**Comment by brandonkelly** [comment]
Sorta related: #6631
